test/controllers/MrnEntryControllerSpec.scala

```diff
@@ -184,4 +192,17 @@ class MrnEntryControllerSpec extends ControllerSpecBase {
     }
   }
 
+  "MrnEntryController on autoFill" should {
+
+    "not clear the mrnPageRefererUrl value in the cache" when {
+      "no value is sent in the request" in {
+        val controller = mrnEntryController(validAnswers.copy(mrnPageRefererUrl = Some(validRefererUrl)))
+        controller.autoFill(mrn)(fakeRequest).futureValue
+
+        theSavedFileUploadAnswers.mrnPageRefererUrl mustBe Some(validRefererUrl)
+      }
+    }
+
+  }
```

Comment:
We don't need to reinstate this test as this is regarding the `mrnPageRefererUrl` parameter that no longer exists.